tools/slo-generator/slo_generator/utils.py

```diff
@@ -82,6 +112,9 @@ def setup_logging():
                         datefmt='%m/%d/%Y %I:%M:%S')
     logging.getLogger('googleapiclient').setLevel(logging.ERROR)
 
+    # Ingore annoying Cloud SDK warning
```

Comment:
Why are we ignoring?

Also, spelling.

Reply:
This pops up when querying `StackdriverBackend` without a service account (i.e authentified with a user using `gcloud auth login`. I believe running with end-users credentials is acceptable when running the `slo-generator` locally.

For `N` configs using the Stackdriver backend, with `M` error budget policy steps, it will pop up `N*M` times, which can quickly pollute all the logs.

Fixed spelling and clarified the comment.